example-client/.env.client.example

```diff
@@ -0,0 +1,38 @@
+# Copy this file to a new file named '.env' to automatically load these
+# values as environment variables when running the client.
+# You will need to fill in the values based on which scenario you are
+# running against:
+# 1. A client testing against a Structure registered locally in the 
```

Comment:
it maybe obvious - but would it be useful to explain what a "client" is?

Reply:
Maybe specify `client program`?

Reply:
check latest